reference/configuration/framework.rst

```diff
@@ -526,10 +526,10 @@ enabled
 
 Whether or not to enable the ``translator`` service in the service container.
 
-fallback
-........
+fallbacks
+.........
 
-**type**: ``string`` **default**: ``en``
+**type**: ``array`` **default**: ``en``
```

Comment:
it allows both a string and an array. Can you please update it to `string|array` ?